monitoring/bin/remove_monitoring_tenant.sh

```diff
@@ -66,4 +74,9 @@ kubectl delete -n $VIYA_NS --ignore-not-found secret prometheus-federate-$VIYA_T
 # Remove non-user-provided certificates
 kubectl delete -n $VIYA_NS --ignore-not-found certificate -l "v4m.sas.com/tenant=$VIYA_TENANT" 2>/dev/null
 
+if [ "$OPENSHIFT_CLUSTER" == "true" ]; then
```

Comment:
Is this the only "OpenShift" specific task needed for removing tenant monitoring?  If so, why have the separate remove_monitoring_tenant_openshift.sh script?  If the other script does "more", why do anything "OpenShift" specific here?